src/utils/os.h

```diff
@@ -0,0 +1,25 @@
+/* date = October 23rd 2022 1:52 pm */
+
+#ifndef OS_H
+#define OS_H
+
+#include "common.h"
+#include "containers/string.h"
+
+#ifdef _WIN32
```

Comment:
Think we can move this in the c file, the header knows nothing about the platform

Reply:
done